tests/unit/generator/test_generator.py

```diff
@@ -145,6 +146,50 @@ def test_get_response_ignore_gapic_metadata():
             assert res.file == CodeGeneratorResponse().file
 
 
+@pytest.mark.parametrize("unversioned_package_disabled", (True, False))
+def test_get_response_ignore_unversioned_package(unversioned_package_disabled):
+    g = make_generator()
+    naming = make_naming(
+        namespace=("Foo",),
+        name="Bar",
+        proto_package="foo.bar.v2",
+        version="v2",
+    )
+    python_settings = client_pb2.PythonSettings(
+        experimental_features=client_pb2.PythonSettings.ExperimentalFeatures(
+            unversioned_package_disabled=unversioned_package_disabled
+        )
+    )
+    service_config = service_pb2.Service(
+        publishing=client_pb2.Publishing(
+            library_settings=[
+                client_pb2.ClientLibrarySettings(
+                    version="foo.bar.v2", python_settings=python_settings
+                )
+            ]
+        )
+    )
+    api = make_api(
+        naming=naming,
+        service_yaml_config=service_config,
+    )
+
+    with mock.patch.object(jinja2.FileSystemLoader, "list_templates") as lt:
```

Comment:
I'd still suggest avoiding short names if possible, though these are two character and use closed to the definition, which makes it more amenabel IMO.

Conider s/`lt`/`lister`/ and s/`gt`/`getter`/

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/2329/commits/6491e53ca0a0a4f5914b705b20778144f0a4588c